libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -111,6 +129,26 @@ class RNBreezSDK: RCTEventEmitter {
             reject(RNBreezSDK.TAG, "Invalid config", nil)
         }
     }
+
+    @objc(signMessage:resolver:rejecter:)
+    func signMessage(_ request: SignMessageRequest, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> SignMessageResponse {
+        do {
+            let response = try BreezSDK.sign_message(message)
+            resolve(response)
+        } catch let error {
+            reject(RNBreezSDK.TAG, "Error calling signMessage", error)
+        }
```

Comment:
This function should convert the reqData into a SignMessageRequest struct, then return a dictionary mapping the SignMessageResponse

```
        if let signMessageRequest = BreezSDKMapper.asSignMessageRequest(reqData: reqData) {
            do {
                let signMessageResponse = try getBreezServices().signMessage(reqData: signMessageRequest)                
                resolve(BreezSDKMapper.dictionaryOf(signMessageResponse: signMessageResponse))
            } catch let err {
                rejectErr(err: err, reject: reject)      
            }
        } else {            
            rejectErr(err: SdkError.Generic(message:"Invalid reqData"), reject: reject)
        }
```